install_dependencies.bash

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+apt-get -qq update
+apt-get -y install lsb-release git devscripts cppcheck debhelper dkms software-properties-common
+
+# Add ppa since kvaser-linlib-dev depends on kvaser-canlib-dev
+apt-add-repository ppa:astuff/kvaser-linux
+apt-get -qq update
```

Comment:
This is not a good idea as this will cause the newly-built `linlib` to rely on the `canlib` that is available in the repository rather than the one you just built. Better to build the new one and manually install the newly-created `.deb` file wth `dpkg` prior to building `linlib`.

Reply:
Thanks for clarifying that, I didn't see any code or anything hinting towards that workflow. My best guess was that you were supposed to wait until the launchpad repo published the new canlib, then run the linlib script after, which would hopefully pull down the new version. I'll fix.

Reply:
Theoretically that works but the process isn't instantaneous. Once you push the new package, you have to first wait for it to be built (which takes a few minutes) then manually deploy it to each supported OS, which also takes time. This is probably something that should also be added to the instructions somewhere.